### DISCUSSION TITLE
SBS (KR) channel links

### DISCUSSION THREAD

**Discussion by AzeemHadzrie** [iptv-org/community#33] [Show and tell]
Hey there, I just want to say I might want to add these channels to the playlist. Here is every playlist I tested:

http://han.ddkdxmkj.com/api/sbs.php?id=1 - SBS
http://han.ddkdxmkj.com/api/sbs.php?id=2 - Nothing else (i mean: V-Power?)
http://han.ddkdxmkj.com/api/sbs.php?id=3 - SBS Plus
http://han.ddkdxmkj.com/api/sbs.php?id=4 - SBS funE
http://han.ddkdxmkj.com/api/sbs.php?id=5 - SBS Golf
http://han.ddkdxmkj.com/api/sbs.php?id=6 - SBS Biz
http://han.ddkdxmkj.com/api/sbs.php?id=7 - SBS Love?
http://han.ddkdxmkj.com/api/sbs.php?id=8 - SBS Power?
http://han.ddkdxmkj.com/api/sbs.php?id=9 - SBS M

**Comment by BellezaEmporium** [comment]
7 Power FM
8 Love FM

**Comment by Alstruit** [comment]
Sorry but the api that generates those links expires 3 minutes.

**Reply by AzeemHadzrie** [reply]
Alright then.

**Reply by AzeemHadzrie** [reply]
SBS M - http://tvlive.sbs.co.kr/sbsmtvpc/sbsmtv1.stream/playlist.m3u8?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE2NjQwMTY5MTUsInBhdGgiOiIvc2JzbXR2MS5zdHJlYW0iLCJkdXJhdGlvbiI6MTgwLCJ1bm8iOiJjZmM5NmIwZS05NjQ1LTQ0ZmEtYWJlNS1lNjFiNjMwNDAyMTUiLCJpYXQiOjE2NjQwMTY3MzV9.sF4PzlaBCFiApoyGBux97il2YZIYahiJyqWiX5pgUTU

**Reply by Alstruit** [reply]
HTTP 403 Error.
